src/public-modules/Authentication/sagas.js

```diff
@@ -5,13 +5,11 @@ import { addressSelector } from 'public-modules/Client/selectors';
 import { actionTypes, actions } from 'public-modules/Authentication';
 import { actionTypes as clientActionTypes } from 'public-modules/Client';
 import { getWeb3Client } from 'public-modules/Client/sagas';
-import { promisify } from 'public-modules/Utilities/helpers';
 import { get } from 'lodash';
 
 const { SET_INITIALIZED } = clientActionTypes;
-const { LOGIN, LOGOUT, GET_CURRENT_USER } = actionTypes;
+const { LOGIN, LOGOUT } = actionTypes;
 const {
-  getCurrentUser: getCurrentUserAction,
```

Comment:
This was redeclared below as an exported function. Perhaps this is a bigger issue that should be solved by itself, instead of me removing this reference?

Reply:
@codeluggage - it's renamed to getCurrentUserAction... so it's safe.

Reply:
There isn't a conflict.

Reply:
@villanuevawill This was warning because it isn't used anywhere. Should it be used somewhere in this file? 
```
  Line 13:  'getCurrentUserAction' is assigned a value but never used  no-unused-vars
```

Reply:
ah if it's not being used - yeah lets remove it :)